int64 num_players
string[4] ip
int64[4] velocity
int64[4] turning

